nextclade/README.md

```diff
@@ -14,6 +14,19 @@ based on [Mutebi et al.][] (J Virol. 2001 Aug;75(15):6999-7008) and
 * Provide the following coloring options on the tree:
   * Genotype assignment from `augur clades`
 
+The clades we annotate (Clade I-VII) are roughly equivalent with the
+following genotypes as described in the aforementioned two papers:
+
+| Clade     | Genotype            |
+|-----------|---------------------|
+| Clade I   | Angola              |
+| Clade II  | East Africa         |
+| Clade III | East Central/Africa |
+| Clade IV  | West Africa I       |
+| Clade V   | West Africa II      |
+| Clade VI  | South America I     |
+| Clade VII | South America II    |
```

Comment:
Thoughts on saving this table as an external TSV that can be referenced here and used by downstream users? It seems like anytime Nextstrain adds it's own clade labeling system, downstream users want an easy way to map it back to the names they are familiar with.

Reply:
I can add an additional TSV and reference it here; I think having the actual table in the README as well is also worthwhile. I dislike duplicating information due to the potential of the two copies getting out of sync, but I don't think the information here is likely to change at all.